src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1484,6 +1545,25 @@ static void ConsoleSettings()
 	ImGui::SameLine();
 	mq::imgui::HelpMarker("This feature allows you to automatically show the MacroQuest Console upon load.");
 
+	if (ImGui::Checkbox("No Titlebar", &s_noTitleBar))
+	{
+		WritePrivateProfileBool("MacroQuest", "NoTitlebar", s_noTitleBar, mq::internal_paths::MQini);
+	}
+
+	if (ImGui::Checkbox("Turn Alpha On", &s_alphaBool))
```

Comment:
This option needs some re-wording and/or clarification as to what it does. Similarly with the option names. "AlphaOn" doesn't really describe meaningfully what it does if it is on or off.

Reply:
This extends to variable names like `s_alphaBool`